src/bindings/python/wheel/setup.py

```diff
@@ -302,27 +313,30 @@ def generate_package(self, src_dirs):
 
         for src_dir in src_dirs:
             local_base_dir = Path(src_dir)
-
-            # skip symlinks of higher level like libX.so or libX.dylib
+            # record real files and its symlinks, and skip symlinks for higher level like libX.so or libX.dylib
```

Comment:
could you please document the logic we have here?
E.g. what we have as an input and what we are trying to achieve